app/src/main/java/com/example/concordia_campus_guide/Database/Converters/StringLongConverter.java

```diff
@@ -0,0 +1,21 @@
+package com.example.concordia_campus_guide.Database.Converters;
+
+import androidx.room.TypeConverter;
+
+public class StringLongConverter {
```

Comment:
is this converter for long types or time types? we should make it clear in the class name